### PR TITLE
Updated environment and lock file

### PR DESCRIPTION
**Pull request by marcus-oscarsson**

### CODE REVIEW

File: conda-environment.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ dependencies:
   - pip
   - poetry
   - nodejs
+  - redis
```
Comment (fabcor-maxiv) [inline conda-environment.yml]: Don't we rather need `redis-server` here?

Comment (marcus-oscarsson) [reply conda-environment.yml]: Hm, yes indeed. I thought it installed the server but it picked up my local server installation, so I did not notice

**Comment by fabcor-maxiv** [comment]
@marcus-oscarsson Do you think you could configure your editor to always put a newline at the end of files? That would be much appreciated -- at least by me :) . When there is no newline at the end of files git puts warnings out, especially for things like `git diff`.

**Comment by marcus-oscarsson** [comment]
No problem :),

Merging this one now, let me know if there is something else you spot and we will try to have a look

**Comment by fabcor-maxiv** [comment]
In order to simplify the installation for development, I thought maybe we could have something like this to `conda-environment-dev.yaml`:

```yaml
dependencies:
  # ...
  - pip:
    - '--editable ../mxcubeweb'
    - '--editable ../mxcubecore'
```

... but I do not know, that seems a bit strange, and I am not even sure that would really work, I have not tested.

**Comment by marcus-oscarsson** [comment]
Hm, that would be very nice. I have no idea if that works either. We can try :)

**Pull request merged**